nav2_bringup/launch/developer_launch.py

```diff
@@ -0,0 +1,225 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is all-in-one launch script intended for use by nav2 developers.
+
+import os
+
+from ament_index_python.packages import get_package_prefix
+from ament_index_python.packages import get_package_share_directory
+from launch.conditions import IfCondition
+from nav2_common.launch import RewrittenYaml
+
+import launch.actions
+import launch_ros.actions
+
+
+def generate_launch_description():
+    # Get the launch directory
+    launch_dir = os.path.join(get_package_share_directory('nav2_bringup'), 'launch')
+
+    # Create the launch configuration variables
+    autostart = launch.substitutions.LaunchConfiguration('autostart')
+    bt_xml_file = launch.substitutions.LaunchConfiguration('bt')
+    map_yaml_file = launch.substitutions.LaunchConfiguration('map')
+    params_file = launch.substitutions.LaunchConfiguration('params')
+    rviz_config_file = launch.substitutions.LaunchConfiguration('rviz_config')
+    simulator = launch.substitutions.LaunchConfiguration('simulator')
+    use_sim_time = text = launch.substitutions.LaunchConfiguration('use_sim_time')
```

Comment:
Is this a typo? It doesn't look used.
```suggestion
    use_sim_time = launch.substitutions.LaunchConfiguration('use_sim_time')
```

Reply:
Yes, some stray cruft. Deleted.